src/main/resources/assets/chisel/lang/de_DE.lang

```diff
@@ -0,0 +1,2774 @@
+
+tile.chisel.concrete_white.name=Weißer Beton
+tile.chisel.concrete_white.cracked.desc.1=Gebrochen
+tile.chisel.concrete_white.bricks-soft.desc.1=Verwitterte Ziegel
+tile.chisel.concrete_white.bricks-cracked.desc.1=Gebrochene Ziegel
+tile.chisel.concrete_white.bricks-triple.desc.1=Breite Ziegel
+tile.chisel.concrete_white.bricks-encased.desc.1=Umhüllte Ziegel
+tile.chisel.concrete_white.braid.desc.1=Verflochten
+tile.chisel.concrete_white.array.desc.1=Aufgestellte Ziegel
+tile.chisel.concrete_white.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_white.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_white.chaotic-medium.desc.1=Ungeordnete Fließen
+tile.chisel.concrete_white.chaotic-small.desc.1=Kleine Ungeordnete Fließen
+tile.chisel.concrete_white.dent.desc.1=Vertiefung
+tile.chisel.concrete_white.french-1.desc.1=Französisch 1
+tile.chisel.concrete_white.french-2.desc.1=Französisch 2
+tile.chisel.concrete_white.jellybean.desc.1=Angeordnet
+tile.chisel.concrete_white.layers.desc.1=Schichten
+tile.chisel.concrete_white.mosaic.desc.1=Mosaik
+tile.chisel.concrete_white.ornate.desc.1=Kunstvoll
+tile.chisel.concrete_white.panel.desc.1=Vertäfelt
+tile.chisel.concrete_white.road.desc.1=Straße
+tile.chisel.concrete_white.slanted.desc.1=Geneigt
+tile.chisel.concrete_white.zag.desc.1=Zackig
+tile.chisel.concrete_white.circularct.desc.1=Kreisförmig
+tile.chisel.concrete_white.circularct.desc.2=Hat CTM
+tile.chisel.concrete_white.weaver.desc.1=Keltisch
+tile.chisel.concrete_white.bricks-solid.desc.1=Ziegel
+tile.chisel.concrete_white.bricks-small.desc.1=Kleine Ziegel
+tile.chisel.concrete_white.circular.desc.1=Kreisförmig
+tile.chisel.concrete_white.tiles-medium.desc.1=Fliesen
+tile.chisel.concrete_white.pillar.desc.1=Säule
+tile.chisel.concrete_white.twisted.desc.1=Verdreht
+tile.chisel.concrete_white.prism.desc.1=Prisma
+tile.chisel.concrete_white.raw.desc.1=Roh
+tile.chisel.concrete_white.emboss.desc.1=Geprägt
+tile.chisel.concrete_white.indent.desc.1=Eingerückt
+tile.chisel.concrete_white.marker.desc.1=Zeichen
+tile.chisel.concrete_white.sunken.desc.1=Versunken
+tile.chisel.concrete_white.poison.desc.1=Gift
+tile.chisel.concrete_white.bricks-chaotic.desc.1=Zertretene Ziegel
+tile.chisel.concrete_white.cuts.desc.1=Schnitte
+
+tile.chisel.concrete_yellow.name=Gelber Beton
+tile.chisel.concrete_yellow.cracked.desc.1=Gebrochen
+tile.chisel.concrete_yellow.bricks-soft.desc.1=Verwitterte Ziegel
+tile.chisel.concrete_yellow.bricks-cracked.desc.1=Gebrochene Ziegel
+tile.chisel.concrete_yellow.bricks-triple.desc.1=Breite Ziegel
+tile.chisel.concrete_yellow.bricks-encased.desc.1=Umhüllte Ziegel
+tile.chisel.concrete_yellow.braid.desc.1=Verflochten
+tile.chisel.concrete_yellow.array.desc.1=Aufgestellte Ziegel
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```

Comment:
You should not translate this since you cannot translate this.

If you do, do something completely unrelated, like `Es ist ein Wunderwerk des Sanitärhandwerks!`